### PR TITLE
os/bluestore: some more cleanups

### PR DESCRIPTION
**Pull request by xiexingguo**

### CODE REVIEW

File: src/os/bluestore/BlueStore.cc [hunk 1]
```diff
@@ -817,10 +809,7 @@ void BlueStore::TwoQCache::_trim(uint64_t onode_max, uint64_t buffer_max)
       Buffer *b = &*p;
       assert(b->is_clean());
       dout(20) << __func__ << " buffer_warm_in -> out " << *b << dendl;
-      assert(buffer_bytes >= b->length);
-      buffer_bytes -= b->length;
-      assert(buffer_list_bytes[BUFFER_WARM_IN] >= b->length);
-      buffer_list_bytes[BUFFER_WARM_IN] -= b->length;
```
Comment (xiexingguo) [inline src/os/bluestore/BlueStore.cc]: We are evicting buffers from the warm_in list here, so the cache_private will be deemed to be BUFFER_WARM_IN. 
Thus the replacement is still equivalent, right?

**Comment by xiexingguo** [comment]
retest this please

File: src/os/bluestore/BlueStore.cc (hunk 1 above)
Comment (liewegas) [inline src/os/bluestore/BlueStore.cc]: Looks like a safe change.  It's just trading performance for maintainability.  I'm ambivalent..

Comment (xiexingguo) [inline src/os/bluestore/BlueStore.cc]: my fault:(,  drop this for now

**Comment by xiexingguo** [comment]
Igor, I bet the performance shall win so I drop the last change.
Thanks for the review.

**Pull request merged**